### DISCUSSION TITLE
Stellar Sorobanathon Submission

### DISCUSSION THREAD

**Discussion by MiriamAndolini** [stellar/sorobanathon#126] [General]
Good Day Stellar Folks! Here are my submissions for the Sorobanathon. Thank you for opening it up to the artists, it was a ton of fun to work on! Please find the links for each of my entries below. 😊

The Quest for the Golden Soroban – [Collection Page](https://nft.fimi.market/detail/collection/sorobanathon)
Prologue – [Just the Essentials](https://nft.fimi.market/detail/nft/Stellar/GA33SRDTEFTXJGGD7BFNWNB6F477N56PTQSDMN36RYRL3DI6CUFLJ467/PRLG)
Chapter 01 – [Hello World ](https://nft.fimi.market/detail/nft/Stellar/GCJKFKVKU3WMOHOCLH3KUNZ736SFQ2YZ6INS4UEKIJIZZ3KVKZGABZUB/CH01)
Chapter 02 – [Zebulon](https://nft.fimi.market/detail/nft/Stellar/GDTPFHAWEZERWVAOZJLLZ2KCD52EXCDMLKKVVX6U2RIUUJMA5XHYQHH7/CH02)
Chapter 03 – [A Single Potion](https://nft.fimi.market/detail/nft/Stellar/GCMGNH7JXLVI3Y7MR3TGNZHFKGQBRHF6GYP3CTGHAWGTLPZ6GEBKA4CS/CH03)
Chapter 04 – [Ozias O'Reilly](https://nft.fimi.market/detail/nft/Stellar/GACOXUG3GCTNJYITBZKVIXTWTEY4JDFCMTEFBHQMYEHHGGELDL2GYZVT/CH04)
Chapter 05 – [The Golden Soroban](https://nft.fimi.market/detail/nft/Stellar/GCWX7OYBR75PFLJFDW4PYP32XC5CV6TUSJQSDSQJZY3IDF6XF3LQOTYF/CH05)
Chapter 06 – [Disappear](https://nft.fimi.market/detail/nft/Stellar/GAFUNT7FQWVIFPP5XWVAAP24Y2R2TBQ6MN55S2V2PHJSM5RRHSFMM5PY/CH06)

All the best,
Miriam

**Comment by stellarjohn** [comment]
Hi Miriam!

Congratulations! Your contribution to has been accepted! 🎉
Your submission may qualify for a reward of 5100 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions.

Cheers!
John

**Reply by MiriamAndolini** [reply]
Thanks John! I’ll get this done 😊🥳
